ush/gefs_ocnice_extractvars.sh

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env bash
+################################################################################
+#   Script:    
+#
+source "${USHgfs}/preamble.sh"
+
+outdirpre=${1}
+varlist=${2}
+dataformat=${3}
+datares=${4}
+datacompress=${5}
+fhout_ocnice=${6}
+cd "${outdirpre}" || true
+
+nh=${FHMIN_GFS}
+while [[ ${nh} -le ${FHMAX} ]];do
+  fnh=$(printf "%3.3d" "${nh}")
+
+  if [[ "${dataformat}" == "grib2" ]];then
+    if [[ ${component_name} == "ocn" ]];then
+      infile=${COM_OCEAN_GRIB}/${datares}/gefs.ocean.${cycle}.${datares}.f${fnh}.grib2
+    fi
+    if [[ ${component_name} == "ice" ]];then
+      infile=${COM_ICE_GRIB}/${datares}/gefs.ice.${cycle}.${datares}.f${fnh}.grib2
+    fi                                                                                                                                                                                                                                   
+    oufile1=${outdirpre}/gefs.${component_name}.${cycle}.${datares}.f${fnh}.grib2
```

Comment:
Since there is only one output file, this should probably be renamed to `outfile`.

Reply:
Done.